### PR TITLE
Added release pipeline

### PR DESCRIPTION
**Pull request by touhidurrr**

### CODE REVIEW

**Comment by lyze237** [comment]
Please adjust the changes I requested, after that I'll merge it.

**Comment by touhidurrr** [comment]
> Please adjust the changes I requested, after that I'll merge it.

Such as?

**Review by lyze237 — CHANGES_REQUESTED** [review]
Check this out

File: .github/workflows/releases.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: "Build & Release"
+
+on: [push, pull_request, workflow_dispatch]
```
[the remark this answers is not in the document]
Comment (touhidurrr) [reply .github/workflows/releases.yml]: fixed with https://github.com/lyze237/GdxJsonConverter/pull/1/commits/80584adf2e4d650888b2997f24d6730fd361e100

**Review by lyze237 — APPROVED** [review]

**Pull request merged**